### PR TITLE
wq: work_queue_graph_workers documentation

### PR DESCRIPTION
**Pull request by sghuang19**
Some documentations for the viz tool.

### CODE REVIEW

**Comment by sghuang19** [comment]
These two are all the documentation locations I found so far.

File: doc/manuals/work_queue/index.md [hunk 1]
```diff
@@ -1907,6 +1907,12 @@ The statistics available are:
 |||
 | manager_load       | In the range of [0,1]. If close to 1, then the manager is at full load <br /> and spends most of its time sending and receiving taks, and thus <br /> cannot accept connections from new workers. If close to 0, the <br /> manager is spending most of its time waiting for something to happen. |
 
+The script `work_queue_graph_workers` is an interactive visualization tool for
+Work Queue transaction logs based on Python `bokeh` package. It can be used to
+visualize the life time of tasks and workers, as well as diagnosing the effects
+of file transfer time on overall performance. See `man
```
Comment (btovar) [inline doc/manuals/work_queue/index.md:1913]: Here lets have a link to man url. (See how it is done in `doc/manuals/manpages`  for https://cctools.readthedocs.io/en/stable/man_pages.)

**Review by btovar — CHANGES_REQUESTED** [review]

File: doc/manuals/work_queue/index.md [hunk 2]
```diff
@@ -1910,8 +1910,9 @@ The statistics available are:
 The script `work_queue_graph_workers` is an interactive visualization tool for
 Work Queue transaction logs based on Python `bokeh` package. It can be used to
 visualize the life time of tasks and workers, as well as diagnosing the effects
-of file transfer time on overall performance. See `man
-work_queue_graph_workers` for detailed information.
+of file transfer time on overall performance. See
+[work_queue_graph_workers(1)](work_queue_graph_workers.md) for detailed
```
Comment (btovar) [inline doc/manuals/work_queue/index.md:1914]: I think you want:
```
[work_queue_graph_workers](../man_pages/work_queue_graphs_workers.md)
```

**Review by btovar — CHANGES_REQUESTED** [review]

**Comment by btovar** [comment]
We are hesitant to update the logs, as it may break tools that process the files downstream...

**Comment by sghuang19** [comment]
But the updates have already been merged. Besides, what we did is just add a line for `TRANSFER`, it shouldn't be conflicting with existing scripts.

File: doc/manuals/work_queue/index.md [hunk 3]
```diff
@@ -1793,17 +1793,18 @@ tasks. It is activated as follows:
...
-# time master_pid WORKER worker_id host:port DISCONNECTION (UNKNOWN|IDLE_OUT|FAST_ABORT|FAILURE|STATUS_WORKER|EXPLICIT)
-# time master_pid WORKER worker_id RESOURCES {resources}
-# time master_pid CATEGORY name MAX {resources_max_per_task}
-# time master_pid CATEGORY name MIN {resources_min_per_task_per_worker}
-# time master_pid CATEGORY name FIRST (FIXED|MAX|MIN_WASTE|MAX_THROUGHPUT) {resources_requested}
-# time master_pid TASK taskid WAITING category_name (FIRST_RESOURCES|MAX_RESOURCES) {resources_requested}
-# time master_pid TASK taskid RUNNING worker_address (FIRST_RESOURCES|MAX_RESOURCES) {resources_allocated}
-# time master_pid TASK taskid WAITING_RETRIEVAL worker_address
-# time master_pid TASK taskid (RETRIEVED|DONE) (SUCCESS|SIGNAL|END_TIME|FORSAKEN|MAX_RETRIES|MAX_WALLTIME|UNKNOWN|RESOURCE_EXHAUSTION) exit_code {limits_exceeded} {resources_measured}
+# time manager_pid MANAGER START|END
```
Comment (btovar) [inline doc/manuals/work_queue/index.md:1796]: Please cleanup this commit, so that only line 1796 is the one that changes.

**Review by btovar — CHANGES_REQUESTED** [review]

**Pull request merged**